### ISSUE TITLE
Transforms: Providing multiple shortcode aliases only matches first one

### ISSUE THREAD

**Issue by jg314** [WordPress/gutenberg#14476]
**Describe the bug**
When you provide multiple shortcode aliases to transform two shortcodes into a single block it looks like only the first shortcode in the array is matched and transformed.

**To Reproduce**
1. Create a block with the following transform:

```javascript
transforms: {
	from: [
		{
			type: 'shortcode',
			tag: [ 'one_time_volunteer_opps', 'flexible_volunteer_opps' ],
			attributes: {
				showOneTime: {
					type: 'boolean',
					shortcode: ( attributes, { shortcode } ) => {
						return shortcode.tag === 'one_time_volunteer_opps' ? true : false;
					},
				}
			},
		},
	]
}
```

2. Add a classic editor block to the page and add `[flexible_volunteer_opps]` into the content.
3. Click "Convert to Blocks" for the classic editor.
4. Even though "flexible_volunteer_opps" is in the `tag` property's array, there is no match and it's not transformed.

**Expected behavior**
I would expect the shortcode to be transformed into the block we created.

**Desktop:**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 72

**Additional context**
- I'm testing this using WordPress 5.1.1.
- On [https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-registration/](https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-registration/) it says "Shortcode tag can also be an array of shortcode aliases", which is why I reached out about this.

Finally, this is my first time posting an issue here so please let me know if there is anything else I can provide that would be helpful.

**Comment by draganescu** [comment]
Howdy @jg314 and welcome to project Gutenberg's GitHub. Thank you for reporting this bug :)

I can confirm from repeating your steps that an array of tags will fail to convert all members, but I have no solution or suggestions. 

Will label this accordingly Needs Testing.

**Comment by jg314** [comment]
Thanks @draganescu. If you can point me to the JavaScript file where the transforms are done I'm happy to take a look and see if I can submit a pull request.

**Comment by mcsf** [comment]
Closed by #17925
